src/compiler/transformers/style-imports.ts

```diff
@@ -95,11 +95,11 @@ const createEsmStyleImport = (
   const importName = ts.factory.createIdentifier(style.styleIdentifier);
   const importPath = getStyleImportPath(transformOpts, tsSourceFile, cmp, style, style.externalStyles[0].absolutePath);
 
-  return ts.createImportDeclaration(
+  return ts.factory.createImportDeclaration(
     undefined,
     undefined,
-    ts.createImportClause(importName, undefined),
-    ts.createLiteral(importPath)
+    ts.factory.createImportClause(false, importName, undefined),
```

Comment:
We add `false` here because the function signature has changed such that `isTypeOnly` is no longer optional, is typed as `boolean`, and is now the first parameter to the function:
```diff
- createImportClause(isTypeOnly: boolean, name: Identifier | undefined, namedBindings: NamedImportBindings | undefined): ImportClause;
+ createImportClause(name: Identifier | undefined, namedBindings: NamedImportBindings | undefined, isTypeOnly?: any): ImportClause;
```